src/styles/main.scss

```
@@ -1,3 +1,68 @@
-body {
+@import './variables';
+
+.body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  counter-reset: section;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  gap: 1px;
+
+
+  &__day {
+    box-sizing: border-box;
+    background-color: $cell-color;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    position: relative;
+    transition: all 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #FFBFCB;
+      transform: translateY(-20px);
+    }
+
+    &::before {
+      counter-increment: section;
+      content: "" counter(section) "";
+      font-family: $font;
+      font-size: 30px;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      margin-right: -50%;
+      transform: translate(-50%, -50%)
+    }
+  }
+
+  @each $day, $value in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $value;
+    }
+  }
```

```suggestion
  @each $day, $value in $days {
    &--start-day-#{$day} :first-child {
      margin-left: ($width + $gap) * $value
    }
  }
```
and calculate margin-left with those vars